src/components/HelperTool.tsx

```diff
@@ -0,0 +1,41 @@
+import * as React from "react";
+import { XMarkIcon } from "@heroicons/react/24/outline";
+import { useState } from "react";
+
+type HelperToolProps = {
+  data: Record<string, any>;
+};
+
+const HelperTool = ({ data }: HelperToolProps) => {
+  const { uid, businessId } = data;
+  const [isOpen, setIsOpen] = useState(true);
+
+  const editUrl = `https://sandbox.yext.com/s/${businessId}/entity/edit3?entityIds=${uid}`;
```

Comment:
We probably don't want to default to sandbox

Reply:
thought is that this will be used mainly for people onboarding through HH, in which case they won't be using prod. Do you have any good ideas on how we could make this configurable? Aaron and I didn't have anything clever

Reply:
Can't this be a local env var and we can preset it to sandbox?

Reply:
Makes sense. What we'll do is set a env var in `.env` called `YEXT_UNIVERSE` and we'll set it to 'sandbox' by default. In future, we can figure out a way to set the universe programmatically.